docs/data/toolpad/studio/index.md

```diff
@@ -4,6 +4,6 @@ title: Toolpad Studio docs - Home
 
 # Toolpad Studio
 
-<p class="description">Toolpad Studio docs</p>
+<p class="description">Toolpad Studio docs.</p>
```

Comment:
We can enforce this in docs-infra: https://github.com/mui/material-ui/pull/44292.